src/linkExposure.js

```diff
@@ -72,7 +72,7 @@ const ignoreSelfLinks = true;
  * @property {matching.MatchPatternSet} linkMatchPatternSet - The match patterns for link URLs.
  * @property {matching.MatchPatternSet} pageMatchPatternSet - The match patterns for pages.
  * @property {boolean} privateWindows - Whether to report exposures in private windows.
- * @property {browser.contentScripts.RegisteredContentScript} contentScript - The content
+ * @property {browser.scripting.RegisteredContentScript} contentScript - The content
```

Comment:
hmm technically this one should stay as it was, since we're still using the old `browser.contentScripts` API in this file

Reply:
I looked into it and they're actually (confusingly) very different objects.
https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/scripting/RegisteredContentScript
https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/contentScripts/RegisteredContentScript

the `contentScripts` one has the `unregister` method built into it, and no other info. the `scripting` seems to be the opposite: lots of info about the script but no built-in method to unregister (you have to call `scripting.unregisterContentScripts` instead)

Reply:
Yeah I'd like to file a followup on this, I don't think WebScience is unregistering these reliably as-is (since I found at least once place it wasn't registering the script).

Reply:
I was pretty confused about this too, I think `scripting.unregisterContentScripts` is the new way to do this. `scripting.getRegisteredContentScripts` can be used to get the list of content scripts so I think all we really need to keep around is the ID that we register the scripts with, we don't need to keep the reference around.

Reply:
Filed https://github.com/mozilla-rally/web-science/issues/309 to follow up on this